app/controllers/concerns/api/pagination.rb

```diff
@@ -1,9 +0,0 @@
-module Api::Pagination
-  extend ActiveSupport::Concern
-
-  included do
-    include Pagy::Backend
-    helper_method :pagy_metadata
-    after_action { pagy_headers_merge(@pagy) if @pagy }
```

Comment:
I think losing the headers would be another breaking change

Reply:
https://github.com/basecamp/geared_pagination#:~:text=headers